website/docs/developer/deployment-guide/creating-orca-archive-bucket.md

```diff
@@ -256,7 +256,7 @@ modifications, which will be detailed below.
       	  "aws:SourceAccount": "000000000000"
         },
       	"ArnLike": {
-      	  "aws:SourceArn": "arn:aws:s3:::PREFIX-orca-reports"
+      	  "aws:SourceArn": ["arn:aws:s3:::PREFIX-orca-archive"]
```

Comment:
any reason why we are changing the bucket name to 
`prefix-orca-archive` from our present `prefix-orca-reports`?

Reply:
It was incorrect. This line should refer to the source of the incoming data rather than the reports storage bucket.

Reply:
oh I see. I thought the bucket name was being changed.